### DISCUSSION TITLE
Stats for nerds

### DISCUSSION THREAD

**Discussion by ALMIGHTY-VC** [fast4x/RiMusic#3522] [Q&A] [answered]
I use modern view i think stats for the nerds is where we long pressed the thumbnail to see info like how much percent is cached, id etc in new version i dont know how to access "stats for nerds" I have enabled it in settings

**Comment by fast4x** [comment, marked as answer]
There is a fix that will be available in 0.6.51

**Reply by ALMIGHTY-VC** [reply]
FIXED!!!
